<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>571</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>464</width>
    <height>491</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>769</width>
    <height>571</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>qStudentMoney</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>792</width>
     <height>578</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pb_ChooseFiles">
        <property name="text">
         <string>Выбрать директорию с данными</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Путь к данным:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_DataPath">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Редактировать таблицы:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pb_editCosts">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Costs.csv</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_editTransport">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Transport.csv</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_editInstitute">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Institute.csv</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_editCaffeCinema">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Caffe-and-cinema.csv</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="4" column="1">
         <widget class="QSpinBox" name="sb_Age">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>130</number>
          </property>
          <property name="value">
           <number>18</number>
          </property>
          <property name="displayIntegerBase">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLineEdit" name="le_Cinema">
          <property name="text">
           <string>SuperCinema</string>
          </property>
          <property name="placeholderText">
           <string>Введите название кинотеатра</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Кафе:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="le_Caffe">
          <property name="text">
           <string>SuperCOffe</string>
          </property>
          <property name="placeholderText">
           <string>Введите название кафе</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Адрес:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Рабочие дни:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="le_City">
          <property name="text">
           <string>Moscow</string>
          </property>
          <property name="placeholderText">
           <string>Введите название города / населённого пункта</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2" alignment="Qt::AlignHCenter">
         <widget class="QCheckBox" name="ch_AgeEffect">
          <property name="text">
           <string>Включить расходы, зависящие от возраста</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="sb_WorkDays">
          <property name="maximum">
           <number>31</number>
          </property>
          <property name="value">
           <number>25</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Месяц:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>ВУЗ:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Возраст:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="le_Address">
          <property name="text">
           <string>Novokosino</string>
          </property>
          <property name="placeholderText">
           <string>Введите название района</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Кинотеатр:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cb_Month">
          <item>
           <property name="text">
            <string>Январь</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Февраль</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Март</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Апрель</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Май</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Июнь</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Июль</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Август</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Сентябрь</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Октябрь</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ноябрь</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Декабрь</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="le_Institute">
          <property name="text">
           <string>MIREA</string>
          </property>
          <property name="placeholderText">
           <string>Введите название ВУЗа</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Город:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="le_Name">
          <property name="placeholderText">
           <string>Введите имя студента</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Имя:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pb_Calculate">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Посчитать</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Студент</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_AnswerName">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>тратит в месяц</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_Answer">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>руб.</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>enableAge(int)</slot>
 </slots>
</ui>
